interbotix_xs_toolbox/interbotix_xs_modules/interbotix_xs_modules/xs_robot/arm.py

```diff
@@ -79,6 +79,8 @@ def __init__(
         logging_level: LoggingSeverity = LoggingSeverity.INFO,
         node_name: str = 'robot_manipulation',
         start_on_init: bool = True,
+        node_owner: bool = True,
+        update_kinematics: bool = True,
```

Comment:
Call this something more descriptive like `update_fk_on_joint_pub`, but maybe less verbose.

Reply:
how about iterative_update_fk

Reply:
Sure